scripts/memory_server.py

```
@@ -0,0 +1,489 @@
+    _db_conn = ensure_db()
+    return _db_conn
+
+
+def _warm_model_async():
+    """Load FastEmbed model in a background thread so server startup is not blocked."""
+    def _warm():
+        try:
+            result = embed_text("warmup")
+            if result and len(result) > 0:
+                _model_ready.set()
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)